capstone/src/main/webapp/about-us.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Us</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="club-name"></div>
+    <div id="navbar"></div>
+    <div id="description"></div>
+    <div id="officers">
+      Officers:
```

Comment:
Can you add tags for "Officers:", "Number of members:", and "Website:"? They can just be <p> tag